portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+    public Comment() {
+
+    }
+
+    public Comment(int id, String name, String description, Date postDate) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.postDate = postDate;
+    }
+
+    private int id;
+    private String name;
+    private String description;
+    private Date postDate;
```

Comment:
My understanding was, that comment might be added to the specific project but I don't see projectId here.

Reply:
Nope, it's a generic comment section. Converting the projects to back-end would be time consuming anyways.

Reply:
Fixed this on #14 ! Please have a look there!